topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,718 @@
+>    - *"Choose computation mode"*: `Bins`
+>       - *"Bin size in bp"*: `1000`
+>       
+>           This corresponds to the length of the fragments that were sequenced; it is not the read length!
+>
+>       - *"Distance between bins"*: `500`
+>
+>           It reduces the computation time for the tutorial
+>
+>    - *"Region of the genome to limit the operation to"*: `chrX`
+>  
+>    Using these parameters, the tool will take bins of 1000 bp separated by 500 bp on the chromosome X. For each bin the overlapping reads in each sample will be computed and stored into a matrix.
+>
+> 4. **plotCorrelation** {% icon tool %} with the following parameters
+>    - {% icon param-files %} *"Matrix file from the multiBamSummary tool"*: `correlation matrix`(output of **multiBamSummary** {% icon tool %})
+>    - *"Correlation method"*: `Pearson`
+>    
+>    Feel free to try different parameters for the configuration of the plot (colors, title, etc)
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> ![Output for plotCorrelation with the correlation scores between the 8 samples](../../images/formation_of_super-structures_on_xi/plotCorrelation.png "Correlation scores between the 8 samples")
+>
+> How are your samples clustered? Does that correspond to your expectations?
+>
+> > ### {% icon solution %} Solution
+> > As one could expect, the input replicates cluster together and the ChIP replicates cluster together. It confirms that the immuno-precipitation step worked on our ChIP replicates.
+> >
+> > We also see that the replicates are clustered together: it is also a good sign that the different experiments (H3K4me3, H3K27me3 or CTCF) seem meaningful.
+> >
+> {: .solution }
+{: .question}
+
+## IP strength estimation
+
+To evaluate the quality of the immuno-precipitation step, we can compute the IP strength. It determines how well the signal in the ChIP-seq sample can be differentiated from the background distribution of reads in the control sample ('input'). After all, around 90% of all DNA fragments in a ChIP experiment will represent the genomic background.
+
+To do that we take the data from the `rep1` of the `wt_H3K4me3` ChIP-seq sample and compare it with its corresponding input sample, using **plotFingerprint** {% icon tool %} of deepTools.
+
+Similar to **multiBamSummary** {% icon tool %}, **plotFingerprint** {% icon tool %} randomly samples genome regions (bins) of a specified length and sums the per-base coverage in the indexed BAM files that overlap with those regions. These coverage values are then sorted according to their rank and the cumulative sum of read counts is plotted.
+
+> ### {% icon hands_on %} Hands-on: IP strength estimation
+>
+> 1. **plotFingerprint** {% icon tool %} with the following parameters
+>    - *"Sample order matters"*: `No`
+>       - {% icon param-files %} *"BAM/CRAM file"*: `wt_input_rep1` and `wt_H3K4me3_rep1`
+>    - *"Region of the genome to limit the operation to"*: `chrX`
```

Comment:
It was written we use down-sampled data. Is it necessary to limit the data to `chrX` or is this not already given by the down-sampling?